great_expectations/checkpoint/checkpoint.py

```diff
@@ -18,6 +18,7 @@
     substitute_template_config,
 )
 from great_expectations.core import RunIdentifier
+from great_expectations.core._docs_decorators import public_api
```

Comment:
Just need to remove this for the unused import checker

Reply:
yup just saw that :)